envoy/api/io_error.h

```diff
@@ -35,6 +35,8 @@ class IoError {
     BadFd,
     // An existing connection was forcibly closed by the remote host.
     ConnectionReset,
+    // Network is unreachable due to network settings.
+    NetUnreach,
```

Comment:
nit: I'd suggest naming this NetworkUnreachable to make the name more readable.

Reply:
done